misk-web/web/packages/@misk/core/src/ducks/index.ts

```diff
@@ -1,9 +1,61 @@
+import { fromJS, List } from "immutable"
 export * from "./simpleDucksUtilities"
 export * from "./routerDucks"
 export * from "./simpleFormDucks"
 export * from "./simpleNetworkDucks"
 
-// from @misk/common/src/actions.ts
+/**
```

Comment:
so what is the API being exposed by this file? Should developers be familiar with the entire file and each type exposed by it? Goes back to the metapoint about making a concise API surface area so we don't overwhelm developers with code to understand

Reply:
Fair point, will address this in a future PR as we formalize the API contracts for other parts of the framework. Will track progress here #177